### PR TITLE
chore(testing): Added minikube detection, updated .env to be sourced

### PR DESCRIPTION
**Pull request by sebbrandt87**
- Added license.lic note to README.md
- wait-for-nexus includes minikube detection
- .env is now sourced instead of xargs exported
- updated pre-commit-config

### CODE REVIEW

File: scripts/.env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-NEXUS_VERSION=3.33.0
-NEXUS_PORT=8081
+export NEXUS_VERSION=3.37.0
```
Comment (anmoel) [inline scripts/.env:1]: please change the documentation of the provider to the new version

Comment (anmoel) [reply scripts/.env:1]: `in file docs/index.md`

Comment (sebbrandt87) [reply scripts/.env:1]: 👍

**Review by stevie- — DISMISSED** [review]
/lgtm

**Review by anmoel — APPROVED** [review]

**Pull request merged**